<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #f2f2f3;
}
/*QLabel{
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}*/

/*QRadioButton{
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
}*/

/*QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(33, 33, 33);
}*/


QLineEdit {
background-color: #f2f2f3; 
border: 0.5px solid gray; 
border-radius: 10%; 
color: black;}
QLabel {
background-color: #f2f2f3; 
border: 0.5px solid gray; 
border-radius: 10%; 
color: black;
}
QPushButton {
background-color: #f2f2f3;
color: black; 
border: 0.5px solid gray; 
border-radius: 10%;
}

QPushButton:pressed {
background-color: #b2b2b2;
color: black; 
border: 0.5px solid gray; 
border-radius: 10%;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonSnapshot">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Snapshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonGif">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveX1">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveX2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveZ2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveZ1">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveY2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMoveY1">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotX1">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotY1">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotX2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotZ2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotZ1">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRotY2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSizeMax">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSizeMin">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choosePointColor">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>750</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>750</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>vertexes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseLineColor">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>780</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>780</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseBackColor">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>810</y>
      <width>100</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>810</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>background</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="GLWidget" name="GLwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPointSize2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPointSize1">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizePoint">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>580</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>680</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLineSize2">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>680</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLineSize1">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>680</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sizeLine">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>680</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonCentrall">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>480</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupProectType</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonParallel">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>480</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupProectType</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonNoPoint">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>550</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupPointDisplay</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonSquare">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>550</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupPointDisplay</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonRound">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>550</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Round</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupPointDisplay</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonSolid">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>650</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupLineType</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonDotted">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>650</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dotted</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupLineType</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButtonNoLine">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>650</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupLineType</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>1139</x>
      <y>40</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>130</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>250</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>370</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>450</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Proection</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>520</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vertexes </string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>620</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>580</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>720</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>40</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>963</x>
      <y>10</y>
      <width>20</width>
      <height>831</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>800</y>
      <width>951</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 50);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <zorder>label_26</zorder>
   <zorder>label_25</zorder>
   <zorder>label_24</zorder>
   <zorder>label_31</zorder>
   <zorder>label_29</zorder>
   <zorder>label_28</zorder>
   <zorder>label_27</zorder>
   <zorder>label_23</zorder>
   <zorder>buttonSnapshot</zorder>
   <zorder>buttonGif</zorder>
   <zorder>buttonMoveX1</zorder>
   <zorder>buttonMoveX2</zorder>
   <zorder>buttonMoveZ2</zorder>
   <zorder>buttonMoveZ1</zorder>
   <zorder>buttonMoveY2</zorder>
   <zorder>buttonMoveY1</zorder>
   <zorder>buttonRotX1</zorder>
   <zorder>buttonRotY1</zorder>
   <zorder>buttonRotX2</zorder>
   <zorder>buttonRotZ2</zorder>
   <zorder>buttonRotZ1</zorder>
   <zorder>buttonRotY2</zorder>
   <zorder>buttonSizeMax</zorder>
   <zorder>buttonSizeMin</zorder>
   <zorder>choosePointColor</zorder>
   <zorder>label_8</zorder>
   <zorder>chooseLineColor</zorder>
   <zorder>label_9</zorder>
   <zorder>chooseBackColor</zorder>
   <zorder>label_10</zorder>
   <zorder>GLwidget</zorder>
   <zorder>buttonPointSize2</zorder>
   <zorder>buttonPointSize1</zorder>
   <zorder>sizePoint</zorder>
   <zorder>label_22</zorder>
   <zorder>buttonLineSize2</zorder>
   <zorder>buttonLineSize1</zorder>
   <zorder>sizeLine</zorder>
   <zorder>radioButtonCentrall</zorder>
   <zorder>radioButtonParallel</zorder>
   <zorder>radioButtonNoPoint</zorder>
   <zorder>radioButtonSquare</zorder>
   <zorder>radioButtonRound</zorder>
   <zorder>radioButtonSolid</zorder>
   <zorder>radioButtonDotted</zorder>
   <zorder>radioButtonNoLine</zorder>
   <zorder>open</zorder>
   <zorder>quit</zorder>
   <zorder>label_30</zorder>
   <zorder>save</zorder>
   <zorder>line</zorder>
   <zorder>plainTextEdit</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupLineType"/>
  <buttongroup name="buttonGroupPointDisplay"/>
  <buttongroup name="buttonGroupProectType"/>
 </buttongroups>
</ui>
